Adds ecs fields for ml jobs (#738)
elastic/stack-docs: docs/en/siem/machine-learning.asciidoc

the main navigation header can be used for for viewing, starting, and stopping
SIEM machine learning jobs.

TIP: To add a custom job to the `Anomaly Detection` interface, add a `SIEM` tag 
to the job's `Group` field ({kib} -> {ml-cap} -> Create/Edit job -> Job 
  details).

[float]
[[included-jobs]]
==== Prebuilt jobs

The SIEM app comes with prebuilt {ml} {anomaly-jobs} for automatically detecting
host and network anomalies. The jobs are displayed in the `Anomaly Detection`
interface. They are available when:

> * You ship data using https://www.elastic.co/products/beats[Beats] and {kib} is 

Reviewer: ```suggestion
* You ship data using https://www.elastic.co/products/beats[Beats], and {kib} is 
```

> * You ship data using https://www.elastic.co/products/beats[Beats] and {kib} is 

Reviewer: ```suggestion
* You ship data using https://www.elastic.co/products/beats[Beats], and {kib} is 
```

configured with the required index patterns (`auditbeat-*`, `filebeat-*`, 
`packetbeat-*`, or `winlogbeat-*` via {kib} -> Management -> Index Patterns).

> * Your shipped data is ECS compliant and {kib} is configured with the data's 

Reviewer: ```suggestion
* Your shipped data is ECS-compliant, and {kib} is configured with the data's 
```

index patterns.

NOTE: Machine learning jobs look back and analyse two weeks of historical data 
prior to the time they are enabled. After jobs are enabled, they continuously 
analyse incoming data. When jobs are stopped and restarted within the two week 
timeframe, previously analysed data is not processed again.

<<prebuilt-ml-jobs>> describes all available {ml} jobs and lists 
which beats are required on your hosts for each job. For information on tuning
anomaly results to reduce the number of false positive, see
<<tuning-anomaly-results>>.

[float]
[[view-anomolies]]
=== View detected anomalies
To view the `Anomalies` table widget and `Max Anomaly Score By Job` details,
the user must have the `ml_admin` or `ml_user` role.

> NOTE: To adjust the `score` threshold for which

Reviewer: ```suggestion
NOTE: To adjust the `score` threshold that controls which
```
